pydis_core/exts/__init__.py

```diff
@@ -1,4 +1,6 @@
 """Reusable Discord cogs."""
-__all__ = []
+from .source import SourceCode
+
+__all__ = [SourceCode]
```

Comment:
proactive magic trailing comma.
```suggestion
__all__ = [
    SourceCode,
]
```